include/erlcloud_ddb2.hrl

```diff
@@ -280,15 +280,15 @@
         {consumed_capacity :: undefined | #ddb2_consumed_capacity{},
          count :: undefined | non_neg_integer(),
          items :: undefined | [erlcloud_ddb2:out_item()],
-         last_evaluated_key :: undefined | erlcloud_ddb2:key(),
+         last_evaluated_key :: undefined | erlcloud_ddb2:key() | binary(),
```

Comment:
I think this must be some more lazy eunit mocking. This should not be a binary.

Reply:
On it.

Reply:
Check eb92f35